package.json

```diff
@@ -12,24 +12,24 @@
     "README.md"
   ],
   "scripts": {
-    "build": "npm run clean && tsc && tsc -m es6 --outDir lib-esm && webpack --env.DEV=no",
-    "clean": "shx rm -rf bundles docs lib lib-esm coverage",
-    "clean:all": "npm run clean && shx rm -rf node_modules package-lock.json && git clean -fX -e '!*.pem'",
+    "build": "npm run clean && tsc && tsc -m es6 --outDir lib-esm && webpack --env DEV=no",
+    "clean": "shx rm -rf ./bundles ./docs ./lib ./lib-esm ./coverage",
+    "clean:all": "npm run clean && shx rm -rf ./node_modules ./package-lock.json && git clean -fX -e \"!*.pem\"",
     "docs": "typedoc",
-    "docs:watch": "watch 'npm run docs' ./src",
-    "lint": "eslint --ignore-path .gitignore '**/*.ts' --fix",
+    "docs:watch": "watch \"npm run docs\" ./src",
+    "lint": "eslint --ignore-path .gitignore \"**/*.ts\" --fix",
```

Comment:
Windows seems to be quirky when it comes to single quotes with globs and files or paths without a leading `./`.

Reply:
👍 - I confirmed that the SDK project now builds properly on Windows